packages/runtime/test-runtime-utils/src/mocksDataStoreContext.ts

```diff
@@ -37,7 +37,7 @@ export class MockFluidDataStoreContext implements IFluidDataStoreContext {
 	public packagePath: readonly string[] = undefined as any;
 	public options: ILoaderOptions = undefined as any;
 	public clientId: string | undefined = uuid();
-	public clientDetails: IClientDetails = undefined as any;
+	public clientDetails: IClientDetails = { capabilities: { interactive: true } };
```

Comment:
Why was this necessary?

Reply:
The new tests I added was failing as it was undefined. I can add the interactive prop to constructor and then use that here.